src/home/index.html

```diff
@@ -8,268 +8,308 @@
-          <p>Language</p>
-          <img src="../assert/images/chevron-h.png" alt="language" />
+      </section>
+      <section class="about">
+        <div class="text-home-about">
+          <h1 class="heading heading-about">About Mwafrika</h1>
+          <h3 class="description-about">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </h3>
+          <h3 class="subdescription">LET'S CONNECT</h3>
+
+          <div class="social-container social-container-about">
+            <ul>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/twitter.png" alt="Twitter" />
+                </a>
+              </li>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/Linkedin.png" alt="LinkedIn" />
+                </a>
+              </li>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/m.png" alt="Medium" />
+                </a>
+              </li>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/github.png" alt="Github" />
+                </a>
+              </li>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/fingers.png" alt="Finger" />
+                </a>
+              </li>
+            </ul>
+            <button type="button" class="btn-tonic">See project</button>
+          </div>
         </div>
-        <div class="langage-list">
+
+        <div class="about-img">
+          <!-- <img src="../assert/images/try.svg" alt="About" /> -->
+          <div class="lang title">
+            <h2>Languages</h2>
+            <img src="../assert/images/chevron-v.png" alt="Language" />
+          </div>
+
           <ul class="list">
-            <li class="lang">
-              <img src="../assert/images/js.png" alt="javascript" />
-              <p>Javascript</p>
+            <li class="back">
+              <a href="#"
+                ><img src="../assert/images/js.png" alt="Javascript"
+              /></a>
+              Javascript
             </li>
-            <li class="lang">
-              <img src="../assert/images/css.png" alt="javascript" />
-              <p>CSS</p>
+            <li class="back">
+              <a href="#"><img src="../assert/images/html.png" alt="HTML" /></a>
+              HTML
             </li>
-            <li class="lang">
-              <img src="../assert/images/html.png" alt="javascript" />
-              <p>HTML</p>
+            <li class="back">
+              <a href="#"><img src="../assert/images/css.png" alt="CSS" /></a>
+              CSS

```

Comment:
- [ ] Please also try to match the Figma design here as closely as possible. notice that the html, CSS, javascript badges have a lighter background.
- [ ]  These badges also have equal widths, notice that your "HTML" and "CSS" badges have different widths than the javascript badge.

|Figma Design|Figma Design|
|---|---|
|![image](https://user-images.githubusercontent.com/18730106/149633734-42be9076-a340-482f-b555-711a4795d0db.png)|![image](https://user-images.githubusercontent.com/18730106/149633718-99436889-3e7d-4440-b5c8-9d2a270970c9.png)|

Reply:
Done!